src/gtest/test_foundersreward.cpp

```diff
@@ -22,9 +24,9 @@
 // The output can be copied into chainparams.cpp.
 // The temporary wallet file can be renamed as wallet.dat and used for testing with zcashd.
 //
-#if 0
+
 TEST(founders_reward_test, create_testnet_2of3multisig) {
-    ECC_Start();
+    //ECC_Start(); this is called on the main class of gtest
```

Comment:
On line 34 bool fFirstRun is created but not initialized; then in on line 36 fFirstRun  is read inside CWallet::LoadWallet.
This is wrong, as we should not read from uninitialized variables. Probably in most most of cases, *nix systems evaluate uninitialized bools to true. @MarcoOl94  please set the right initial value